CHANGELOG.md

```diff
@@ -207,7 +214,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Initial release to rubygems.org
 
 
-[Unreleased]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v5.0.1...HEAD
+[Unreleased]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v5.0.2...HEAD
+[5.0.2]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v5.0.1...v5.0.2
```

Comment:
these links don't work. Since version 4.1.3 there is a `.freeze` suffix

Reply:
Added `.freeze` to all links beyond 4.1.3